pkg/common/proto/agent.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
+
+service AgentService{
+    rpc PerformFantasyActions(FantasyLandscape) returns (FantasySelections) {}
+}
+
+message FantasyLandscape{
```

Comment:
We'll want a date in here somewhere I think.  Like we'll want to run simulations and say "assume you're on this date, what do you do?"

Reply:
i added this matchNumber which is a little more generic I think since it will allow for scenarios where a game is on Friday for some games vs. Sunday for others.